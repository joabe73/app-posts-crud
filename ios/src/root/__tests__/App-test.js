import React from 'react'
import { create } from 'react-test-renderer'

import Root from '../../containers/initial/initial.view'

const controller = {
  email: '', 
  handleEmail: () =>{}, 
  handleLogin: () =>{}, 
  loading: false, 
  loginError: ''
}

describe('sould render root module', () => {
  it('renders correctly', () => {
    const componet = create(<Root controller={controller} />)
    const tree = componet.toJSON()
    expect(tree).toMatchSnapshot()
  });
})

