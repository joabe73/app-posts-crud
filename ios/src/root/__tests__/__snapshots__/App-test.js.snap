// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sould render root module renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#FFFFFF",
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "justifyContent": "center",
          "paddingBottom": 20,
          "paddingLeft": 20,
          "paddingRight": 20,
          "paddingTop": 20,
        },
      ]
    }
  >
    <Text
      align="center"
      color="#1a6bed"
      style={
        Array [
          Object {
            "color": "#1a6bed",
            "fontSize": 20,
            "fontWeight": "normal",
            "lineHeight": 32,
            "textAlign": "center",
          },
        ]
      }
      theme={
        Object {
          "text": "#000000",
        }
      }
      title={true}
      weight="normal"
    >
      Posts Air Liquide
    </Text>
    <Text
      align="center"
      color="#a6a6a6"
      helper={true}
      style={
        Array [
          Object {
            "color": "#a6a6a6",
            "fontSize": 14,
            "fontWeight": "normal",
            "lineHeight": 20,
            "textAlign": "center",
          },
        ]
      }
      theme={
        Object {
          "text": "#000000",
        }
      }
      weight="normal"
    >
      para interagir é só informar o email cadastrado
    </Text>
    <View
      autoFocus={true}
      onChangeText={[Function]}
      style={
        Array [
          Object {
            "alignSelf": "stretch",
            "marginTop": 10,
            "width": "100%",
          },
        ]
      }
      value=""
    >
      <TextInput
        allowFontScaling={true}
        autoFocus={true}
        height={false}
        onChangeText={[Function]}
        rejectResponderTermination={true}
        style={
          Array [
            Object {
              "backgroundColor": "#FFFFFF",
              "borderColor": "#cccccc",
              "borderWidth": 1,
              "height": 40,
              "marginTop": 5,
              "paddingBottom": 8,
              "paddingLeft": 8,
              "paddingRight": 8,
              "paddingTop": 8,
              "width": "100%",
            },
          ]
        }
        underlineColorAndroid="transparent"
        value=""
      />
      <Text
        align="left"
        color="#cc0000"
        helper={true}
        style={
          Array [
            Object {
              "color": "#cc0000",
              "fontSize": 14,
              "fontWeight": "normal",
              "lineHeight": 20,
              "textAlign": "left",
            },
          ]
        }
        theme={
          Object {
            "text": "#000000",
          }
        }
        weight="normal"
      />
    </View>
    <View
      height={22}
      style={
        Array [
          Object {
            "height": 22,
            "width": 0,
          },
        ]
      }
    />
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "transparent",
          "borderColor": "#1a6bed",
          "borderRadius": 32,
          "borderWidth": 1,
          "flexDirection": "row",
          "justifyContent": "center",
          "opacity": 0.4,
          "paddingBottom": 4,
          "paddingLeft": 12,
          "paddingRight": 12,
          "paddingTop": 4,
          "shadowColor": "gray",
          "shadowOffset": Object {
            "height": 0,
            "width": 0,
          },
          "shadowOpacity": 0.5,
          "shadowRadius": 4,
        }
      }
    >
      <Text
        align="left"
        color="#1a6bed"
        size={16}
        style={
          Array [
            Object {
              "color": "#1a6bed",
              "fontSize": 16,
              "fontWeight": "bold",
              "lineHeight": 24,
              "textAlign": "left",
            },
          ]
        }
        theme={
          Object {
            "text": "#000000",
          }
        }
        weight="bold"
      >
        Entrar
      </Text>
    </View>
  </View>
</View>
`;
